alpine-rails-nginx/etc/runit/1

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 
-set -o errexit -o pipefail -o noglob -o noclobber -o nounset
-IFS=$'\n\t'
+source strict-mode.sh
```

Comment:
Yes, it's copied into `/usr/local/sbin` in https://github.com/dkubb/dockerfiles/blob/master/alpine-ruby/Dockerfile#L17